### PR TITLE
Add perl-complete-common

### PR DESCRIPTION
**Pull request by xuzhougeng**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by xuzhougeng** [comment]
@conda-forge/staged-recipes ready for review!

**Comment by xuzhougeng** [comment]
@conda-forge/help-perl The recipe has passed all the checks. Kindly make a review. Thanks.

**Comment by jakirkham** [comment]
@conda-forge/bioconda-recipes ?

File: recipes/perl-complete-common/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  imports:
+    - Complete::Common
+
+  # You can also put a file called run_test.pl (or run_test.py) in the recipe
+  # that will be run at test time.
+
+about:
+  home: https://metacpan.org/release/Complete-Common
+  license: GPL-1.0-or-later
+  license_file: LICENSE.txt
```
Comment (mbargull) [inline recipes/perl-complete-common/meta.yaml:35]:
```suggestion
  license: GPL-1.0-or-later OR Artistic-1.0-Perl
  license_file: LICENSE
```
The packages already contains a `LICENSE` file which we can reuse. Please also remove the `LICENSE.txt` from the recipe directory.

**Comment by mbargull** [comment]
> Recipe with the same name exists in bioconda: please discuss with @conda-forge/bioconda-recipes.

Moving this to conda-forge is appreciated!

**Pull request closed without merging**